src/test/java/com/blocklynukkit/wasm/tests/MoudleTest.java

```diff
@@ -0,0 +1,165 @@
+package com.blocklynukkit.wasm.tests;
```

Comment:
Is this class accidentally added? If you intend to include this test as well as other changes, please rename the package name to align with other classes :)

Reply:
Sorry I forgot to remove my tests. I'll delete them soon.